src/components/AmountTextInput.tsx

```diff
@@ -57,6 +57,8 @@ function AmountTextInput(
             role={CONST.ROLE.PRESENTATION}
             onKeyPress={onKeyPress as (event: NativeSyntheticEvent<TextInputKeyPressEventData>) => void}
             touchableInputWrapperStyle={touchableInputWrapperStyle}
+            autoCorrect={false}
+            spellCheck={false}
```

Comment:
Maybe let's mention iPad in the comment? That's the actual only reason we add these properties.

Reply:
Added a comment